Lib/ufo2ft/makeotfParts.py

```diff
@@ -156,6 +154,19 @@ def setupAnchorPairs(self):
 
         self.mkmkAnchorPairs = self.anchorPairs
 
+    def _findLayoutFeatures(self):
+        """Returns what OpenType layout feature tags are present in the UFO."""
+        if self.font.path is None:
+            return set()
+        feapath = os.path.join(self.font.path, "features.fea")
+        if not os.path.exists(feapath):
+            return set()
+        glyphMap = self.outline.getReverseGlyphMap()
+        parser = feaLib.parser.Parser(feapath, glyphMap=glyphMap)
```

Comment:
I guess it would be nice to reuse the ast generated here to compile the features rather than redo it (in setupFile_featureTables), but maybe it's not worth the trouble?